nix/modules/holochain.nix

```diff
@@ -13,6 +13,7 @@
       commonArgs = {
         RUST_SODIUM_LIB_DIR = "${pkgs.libsodium}/lib";
         RUST_SODIUM_SHARED = "1";
+        LIBCLANG_PATH="${pkgs.llvmPackages.libclang.lib}/lib";
```

Comment:
Even though clang is already provided, the libraries aren't found and used unless you tell the build where to find them. This is only required in Nix as far as I can tell.